code/game/objects/structures/kitchen_spike.dm

```diff
@@ -88,69 +51,71 @@
-			H.adjustBruteLoss(20)
-			return
+		if(istype(G.affecting, /mob/living/))
+			if(!buckled_mob)
+				if(do_mob(user, src, 50))
+					if(buckled_mob) //to prevent spam/queing up attacks
+						return
+					if(G.affecting.buckled)
+						return
+					var/mob/living/H = G.affecting
+					playsound(src.loc, "sound/effects/splat.ogg", 25, 1)
+					H.visible_message("<span class='danger'>[user] slams [G.affecting] onto the meat spike!</span>", "<span class='userdanger'>[user] slams you onto the meat spike!</span>", "<span class='italics'>You hear a squishy wet noise.</span>")
+					H.loc = src.loc
+					H.emote("scream")
+					if(istype(H, /mob/living/carbon/human)) //So you don't get human blood when you spike a giant spidere
+						var/turf/pos = get_turf(H)
+						pos.add_blood_floor(H)
+					H.adjustBruteLoss(30)
+					H.buckled = src
+					H.dir = 2
+					buckled_mob = H
+					var/matrix/m120 = matrix()
+					m120.Turn(180)
+					animate(H, transform = m120, time = 3)
+					H.pixel_y = H.get_standard_pixel_y_offset(180)
+					return
 		user << "<span class='danger'>You can't use that on the spike!</span>"
 		return
 	..()
 
-///obj/structure/kitchenspike/MouseDrop_T(var/atom/movable/C, mob/user)
-//	if(istype(C, /obj/mob/carbon/monkey)
-//	else if(istype(C, /obj/mob/carbon/alien) && !istype(C, /mob/living/carbon/alien/larva/slime))
-//	else if(istype(C, /obj/livestock/spesscarp
+/obj/structure/kitchenspike/user_buckle_mob(mob/living/M, mob/living/user) //Don't want them getting put on the rack other than by spiking
+	return
+
+/obj/structure/kitchenspike/user_unbuckle_mob(mob/living/carbon/human/user)
+	if(buckled_mob && buckled_mob.buckled == src)
+		var/mob/living/M = buckled_mob
+		if(M != user)
+			M.visible_message(\
+				"[user.name] tries to pull [M.name] free of the [src]!",\
+				"<span class='notice'>[user.name] is trying to pull you off the [src], opening up fresh wounds!</span>",\
+				"<span class='italics'>You hear a squishy wet noise.</span>")
+			if(!do_after(user, 300, target = src))
+				if(M && M.buckled)
+					M.visible_message(\
+					"[user.name] fails to free [M.name]!",\
+					"<span class='notice'>[user.name] fails to pull you off of the [src].</span>")
+				return
 
-/obj/structure/kitchenspike/attack_hand(mob/user)
-	if(..())
-		return
-	if(occupied)
-		if(meat >= 1)
-			new meattype(src.loc)
-			meat--
-			if(meat > 1)
-				usr << "<span class='notice'>You remove some meat from [src].</span>"
+		else
+			M.visible_message(\
+			"<span class='warning'>[M.name] struggles to break free from the [src]!</span>",\
+			"<span class='notice'>You struggle to break free from the [src], exacerbating your wounds! (Stay still for two minutes.)</span>",\
+			"<span class='italics'>You hear a wet squishing noise..</span>")
+			M.adjustBruteLoss(30)
+			if(!do_after(M, 1200, target = src))

```

Comment:
@RemieRichards 

Was considering making it take longer than a stun to put someone on as well, so it'd require two people or cuffing them first (at which point you could have killed them anyway).